pkg/platform/model/buildplanner/buildplanner.go

```diff
@@ -20,32 +14,6 @@ type client struct {
 	gqlClient *gqlclient.Client
 }
 
-func logRequestVariables(req gqlclient.Request) {
-	if !strings.EqualFold(os.Getenv(constants.DebugServiceRequestsEnvVarName), "true") {
-		return
-	}
-
-	vars, err := req.Vars()
-	if err != nil {
-		// Don't fail request because of this errors
-		logging.Error("Failed to get request vars: %s", err)
-		return
-	}
-
-	for _, v := range vars {
-		if _, ok := v.(*buildexpression.BuildExpression); !ok {
-			continue
-		}
-
-		beData, err := json.MarshalIndent(v, "", "  ")
-		if err != nil {
-			logging.Error("Failed to marshal build expression: %s", err)
-			return
-		}
-		logging.Debug("Build Expression: %s", string(beData))
-	}
-}
```

Comment:
I've dropped this because we're no longer sending *buildexpression.BuildExpression. Instead we simply send the byte representation. This is because we want to keep build expressions internal to buildscripts as much as possible.

I've not replaced this because you can already get this data by logging API requests.